README.md

```diff
@@ -37,21 +37,26 @@ $3T(\frac{n}{3}) + n^5$ otherwise <br>
 
 Solve by substitution 
 
-$T(n) = 3T(\frac{n}{3}) + n^5$<br>
-$T(n) = 3(3T(\frac{n/3}{3}) + (\frac{n}{3})^5) + n^5$<br>
-$T(n) = 3^2T(\frac{n}{3^2}) + \frac{n^5}{3^4} + n^5$<br>
-$T(n) = 3^3T(\frac{n}{3^3}) + \frac{n^5}{3^5} + \frac{n^5}{3^4} + n^5$<br>
-
-Looking at the pattern, we can see the recurrence relation is
-
-$T(n) = 3^iT(\frac{n}{3^i}) + \displaystyle\sum_{j=0}^i \frac{n^5}{3^{i+2}}  + n^5$.<br>
-To terminate the recurrence, $T(n)$ must be <= 1. <br>
-According to the pattern, $T(1) = T(\frac{n}{3^i})$<br>
-$\frac{n}{3^i} = 1$<br>
-$n = 3^i$<br>
-$i = \log_{3} n$ <- value of i when n = 1. <br>
-When we plug it back into the equation we get <br>
-$3^{\log_{3} n}  * T(1) + \displaystyle\sum_{j=0}^{\log_{3} n} \frac{n^5}{3^{(\log_{3} n) + 2}} + n^5$<br>
-Which simplifies to $n + \displaystyle\sum_{j=0}^{\log_{3} n}\frac{n^5}{n+2} + n^5$<br>
-
-so the time complexity is $\Theta(n^5)$
+$T(n)=3T(\frac{n}{3})+n^5+C$
```

Comment:
Where does the $+C$ come from?

Reply:
Sorry I will add it in original relation